src/common/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+set(ICEBERG_COMMON_SOURCES util/string_builder.cc status.cc)
+set(ICEBERG_COMMON_INCLUDES "${ICEBERG_API_DIR}")
+
+ADD_ICEBERG_LIB(
+  iceberg_common
```

Comment:
Why do we need a `iceberg_common` library? Isn't `iceberg-core` the common stuff?

Reply:
> Why do we need a `iceberg_common` library? Isn't `iceberg-core` the common stuff?

I was thinking that maybe puffin also need the abilities that this module provide? I'm fine if you think this should be moved to iceberg-core.